qa/integration/specs/logstash_to_logstash_spec.rb

```diff
@@ -51,10 +44,17 @@ def get_temp_path_dir
   end
 
   def run_logstash_instance(config_name, options = {})
-    api_port = 9600 + rand(1000)
```

Comment:
random allocation means occasional collisions.

I've elected to keep track of an offset in a range of 1000, looping back to 0 if we ever have enough tests to do so.


instead we maintain an offset and use the next one, looping when we hit 1000 (we won't).